python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -113,6 +113,13 @@ def test_lock(self):
         self.assertEqual(msg, errors.next()[2])
         self.assertNoMoreErrors(errors)
 
+    def test_file_list(self):
+        file_list = tidy.get_file_list('./python/tidy/servo_tidy_tests/test_ignored', only_changed_files=False, exclude_dirs=[])
```

Comment:
Shall we have a local variable `base_path = ./python/tidy/servo_tidy_tests/test_ignored`?